picker/Rofimoji.py

```diff
@@ -75,8 +74,7 @@ def __init__(self) -> None:
     def parse_arguments(self) -> argparse.Namespace:
         parser = configargparse.ArgumentParser(
             description='Select, insert or copy Unicode characters using rofi.',
-            default_config_files=[os.path.join(directory, 'rofimoji.rc') for directory in
-                                  BaseDirectory.xdg_config_dirs]
+            default_config_files=[directory / 'rofimoji.rc' for directory in BaseDirectory.xdg_config_dirs]
```

Comment:
`BaseDirectory.xdg_config_dirs` doesn't return `List[Path]` for me, just `str` :/

Reply:
Sorry, I probably misread the `pyxdg` docs. Will fix it here and maybe propose the change upstream.